notebooks/23.Out-of-core_Learning_Large_Scale_Text_Classification.ipynb

```diff
@@ -752,7 +752,7 @@
    "source": [
     "from sklearn.linear_model import SGDClassifier\n",
     "\n",
-    "sgd = SGDClassifier(loss='log', random_state=1)\n",
```

Comment:
why not the default value of 100? How slow is it?

Reply:
The warning told me that it should be 5 :)

http://scikit-learn.org/dev/modules/generated/sklearn.linear_model.SGDClassifier.html

It seems that from 0.21 it will be 1000. We can put it at that value.

Reply:
no warning told you ``n_iter`` will be ``5``, or rather ``max_iter`` with ``tol=0`` ;)